db.go

```diff
@@ -149,7 +149,8 @@ func (db *DB) AddMessage(filename string) (*Message, error) {
 	defer C.free(unsafe.Pointer(cfilename))
 
 	var cmsg *C.notmuch_message_t
-	if err := statusErr(C.notmuch_database_add_message(db.toC(), cfilename, &cmsg)); err != nil {
+	var copts = C.notmuch_database_get_default_indexopts(db.toC())
+	if err := statusErr(C.notmuch_database_index_file(db.toC(), cfilename, copts, &cmsg)); err != nil {
```

Comment:
The comments in `notmuch.h` says that add_message passes `NULL` here; rather than getting the options manually, we should do the same.

It's worth nothing that this version actually has a memory leak, since the options are never freed. Passing `nil` will fix this.

Reply:
Ah brilliant, that's resolved a segfault I saw. I had convinced myself it was in my application code. Not sure how I missed the comment in `notmuch.h` before.

Thanks so much for your help in spotting that :) :+1:

Reply:
No problem. Merged.